app/i18n/index.js

```diff
@@ -12,9 +12,10 @@ const languages = {
     hu: { translation: require('./lang/hu.json') },
     it: { translation: require('./lang/it.json') },
     nl: { translation: require('./lang/nl.json') },
-    pt: { translation: require('./lang/pt-br.json') },
+    ptBR: { translation: require('./lang/ptBR.json') },
```

Comment:
Why are you making this change?

Reply:
https://github.com/jitsi/jitsi-meet/tree/master/lang

I think it would be better for both sides to use the same one.

"pt": "Portuguese",
"ptBR": "Portuguese (Brazil)",

Reply:
Please dont put this unrelated changes into the chinese translation PR. 

In addition ptBR is not a proper language code, it would be pt-BR, as this is chosen by navigator.language, see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language